data/core/chitchat.md

```diff
@@ -425,9 +441,9 @@
     - slot{"onboarding": true}
     - utter_built_bot_before
 * deny
-    - utter_explain_stack
-    - utter_stack_details
-    - utter_explain_nlucore
+    - utter_explain_rasa_components
+    - utter_rasa_components_details
+    - utter_ask_explain_nlucorex
 * how_to_get_started{"product": "core"}
     - utter_explain_core
     - utter_also_explain_nlu
```

Comment:
So to be clear, if they say they want to get started with everything, we explain nlu, core, and x, but if they say they want to get started with nlu we explain nlu and core and vice versa (but not x) -- is this intended?

Reply:
just noticed below that when you explain x, you also explain NLU and Core, so this is the only case where everything is not explained

Reply:
Core and NLU seem like two siblings that can't live without each other. So if they're asking about one of Core/NLU, we might be serving to unnecessarily extending the conversation by offering to about Rasa X.